docs/admin/support.rst

```diff
@@ -1,37 +1,37 @@
 Getting support for Weblate
 ===========================
 
-Weblate is a copylefted libre software with community support. Subscribers
-receive priority support at no extra charge. Prepaid help packages are
-available for everyone. You can find more information about current support
+
+Weblate is copylefted libre software with community support.
+Subscribers receive priority support at no extra charge. Prepaid help packages are
+available for everyone. You can find more info about current support
 offerings at <https://weblate.org/support/>.
 
 .. _activate-support:
 
-Activating support
+Integrating support
 ------------------
 
-Since Weblate 3.8 the purchased support packages can be activated in a Weblate
-management interface. The activation will enable peridic registration on
```

Comment:
I was thinking "Doing so registers your instance on official Weblate servers.", but I don't know what it means…?

Reply:
It periodically sends basic details to the server to check support status. The server can also respond with configuration for some additional services. Currently, the only additional service is backup, see https://docs.weblate.org/en/latest/admin/backup.html#cloudbackup